testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -373,6 +373,12 @@ impl RunLoop {
                     "Synchronized burnchain up to block height {} (chain tip height is {})",
                     block_height, burnchain_height
                 );
+            } else if ibd {
+                // drive block processing after we reach the burnchain tip.
+                // we may have downloaded all the blocks already,
+                // so we can't rely on the relayer alone to
+                // drive it.
+                coordinator_senders.announce_new_stacks_block();
```

Comment:
I'm curious what scenario this is addressing -- `net::relay` calls this every time a Stacks block is received (which isn't driven by `node.relay_sortition_notify()`). How could we end up in a scenario where a Stacks block was received but the coordinator needs to be woken up again?

Reply:
I don't know.  All I know is that without this, I'm able to boot up a Stacks node and have it reach the burn chain tip, but then stall out when processing Stacks blocks.  It has all of them -- it downloaded them all -- but it just stops processing them.  It only resumes processing them once a new Bitcoin block is processed.

Reply:
Ah, okay -- this probably needs to be invoked in #2370's "do not mine" branch as well, then.

Reply:
Actually, I think I figured this out.  The node downloads microblock streams separately from blocks (it must do so, because it needs both the parent and child block to validate the microblock stream).  This means that in practice, the node spends the latter portion of its boot-up sequence downloading just microblock streams.  But, these don't trigger a Stacks block announcement in the relayer.  I've since patched it to do this.

Reply:
Ah, yes, that makes sense